lib/eslint/eslint.js

```diff
@@ -766,6 +768,17 @@ class ESLint {
         return createRulesMeta(resultRules);
     }
 
+    /**
+     * Indicates if the given feature flag is enabled for this instance.
+     * @param {string} flag The feature flag to check.
+     * @returns {boolean} `true` if the feature flag is enabled, `false` if not.
+     */
+    hasFlag(flag) {
+
+        // note: Linter does validation of the flags
+        return privateMembers.get(this).linter.hasFlag(flag);
+    }
```

Comment:
Not necessarily. For instance, there could be an `unstable_config_lookup` flag that becomes inactive when the feature is finished, at which point we introduce a `v10_config_lookup` to indicate it will become on-by-default in v10.